entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -l
+
+cp /evaluator.js /github/workspace
+npm install -g jest
+jest --json --outputFile=evaluation.json
+node evaluator.js evaluation.json requirements_mapping.json result.json
```

Comment:
Will the results file be generated even when running locally?

Reply:
I was thinking about it and it might be better to set the evaluation as the output when running locally and set the result as the output when running inside the github action, resulting in two outputs.

This is because when creating the tests we need to see if they failed and why, to understand if the test must be fixed or not. But I don't know if the Jest evaluator also needs it, or if only the feature tests made with capybara needs it. What's your opinion?

Reply:
But if I just console in development mode, I never test the POST request. There is a code in this project select the URL to send the POST. So we could test all the steps in locally.

Reply:
That makes sense, I'm just worried about polluting the evaluations when creating or fixing tests. Maybe a way to check all the steps is to create a sample repo or something like it.

Reply:
Actually we could just run the Forest Admin locally. My bad! :sweat_smile: